src/compiler/lexer/Compiler_lexer.cpp

```diff
@@ -348,6 +350,14 @@ Token *Lexer::parseSyntax(Token *start_token, Tokens *tokens)
 		case LeftBracket: case LeftParenthesis:
 		case ArrayDereference: case HashDereference: case ScalarDereference:
 		case ArraySizeDereference: {
+			// Syntax error, It didn't close the brackets.
+			if (pos+1 == tokens->end()) {
+				fprintf(stderr, 
```

Comment:
We needs reconsider error handling. if syntax error occurred, `tokenize()` 
should return error value and not force exit program.

Reply:
I'm thinking it is like Go's error handling so, I agreed it.